engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
@vrozov  there is a difference as I explained earlier.  The CUSTOM_SSL_KEYSTORE_CONFIG  setting used in this JIRA is only processed by the Apex CLI (Stram client) and so it doesn't need to be an attribute that any code on the App Master side needs to be aware of.

As far as usability or user confusion goes, I think a setting/property is preferable to an attribute from usability point of view so @devtagare decided to use a setting here and I agree with that decision and personally I don't believe it is confusing.